waku.md

```diff
@@ -75,27 +75,27 @@ light-node      = BIT
 waku-envelope   = "[" expiry ttl topic data nonce "]"
 
 ; 4 bytes (UNIX time in seconds)
-expiry          = 4*OCTET
+expiry          = 4OCTET
 
 ; 4 bytes (time-to-live in seconds)
-ttl             = 4*OCTET
+ttl             = 4OCTET
 
 ; 4 bytes of arbitrary data
-topic           = 4*OCTET
+topic           = 4OCTET
 
 ; byte array of arbitrary size 
 ; (contains encrypted message)
 data            = OCTET
 
 ; 8 bytes of arbitrary data 
 ; (used for PoW calculation)
-nonce           = 8*OCTET
+nonce           = 8OCTET
 
-messages        = *waku-envelope
+messages        = 1*waku-envelope
 
 p2p-request     = waku-envelope
 
-p2p-message     = waku-envelope
+p2p-message     = 1*waku-envelope
```

Comment:
Is it encoded  differently? IIRC it is either waku-envelope OR an rlp list of waku envelopes. @adambabik ?